docs/snippets/full-external-secret.yaml

```diff
@@ -93,30 +93,11 @@ spec:
   # Used to fetch all properties from the Provider key
   # If multiple dataFrom are specified, secrets are merged in the specified order
   dataFrom:
-  - extract:
-      key: database-credentials
-      version: v1
-      property: data
-      conversionStrategy: Default
-      decodingStrategy: Auto
-    rewrite:
```

Comment:
Hey :wave: why do you want to remove all the `extract` / `find` use-cases and replace them with the `sourceRef.generatorRef`? I think it would be valuable to have both in this example.

Reply:
Hey moolen, I had understood from the issue that the ideal would be to remove the extract/find. But really, both cases in this example would certainly be more valuable. I'm already working to add them.